components/brave_wallet/browser/fil_response_parser_unittest.cc

```diff
@@ -62,4 +62,78 @@ TEST(FilResponseParserUnitTest, ParseFilGetTransactionCount) {
   EXPECT_FALSE(brave_wallet::ParseFilGetTransactionCount(json, &value));
 }
 
+TEST(FilResponseParserUnitTest, ParseFilEstimateGas) {
+  std::string json =
+      "{\"jsonrpc\":\"2.0\",\"result\":{\"Version\":0,\"To\":"
```

Comment:
Use R string and format it for readability in this file.

Reply:
done

Reply:
https://github.com/brave/brave-core/pull/12896#pullrequestreview-939858256